### PR TITLE
[Fixtures] Improve fixtures.

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         |master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

Screenshot 2019-07-18 at 12 29 10
Screenshot 2019-07-18 at 12 30 27

todo:

- [x] fix prices
- [x] product name with the relevant photo
- [ ] fix variant generate unnesessary picture 
- [ ] Taxes
- [x] Imrove Variants name

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Fixture/NewTshirtProductFixture.php [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                        ],
+                        [
+                            'code' => 'womens_t_shirts',
+                            'translations' => [
+                                'en_US' => [
+                                    'name' => 'Women',
+                                    'slug' => 't-shirts/women',
+                                ],
+                                'fr_FR' => [
+                                    'name' => 'femme',
```
Comment (CoderMaggie) [inline src/Sylius/Bundle/CoreBundle/Fixture/NewTshirtProductFixture.php]:
```suggestion
                                    'name' => 'Femme',
```

File: src/Sylius/Bundle/CoreBundle/Fixture/NewTshirtProductFixture.php [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        for ($i = 0; $i < $options['amount']; ++$i) {
+            $categoryTaxonCode ='womens_t_shirts';
+
+            $products[] = [
+                'name' =>  sprintf('%s-w', $productsNames[$i]),
+                'code' => $this->faker->uuid,
+                'main_taxon' => $categoryTaxonCode,
+                'taxons' => ['t_shirts', $categoryTaxonCode],
+                'product_attributes' => [
+                    't_shirt_brand' => $this->faker->randomElement(['Nike', 'Adidas', 'JKM-476 Streetwear', 'Potato', 'Centipede Wear']),
```
Comment (CoderMaggie) [inline src/Sylius/Bundle/CoreBundle/Fixture/NewTshirtProductFixture.php]:
```suggestion
                    't_shirt_brand' => $this->faker->randomElement(['You are breathtaking', 'Modern Wear', 'Celsius Small', 'Date & Banana']),
```

:)

File: src/Sylius/Bundle/CoreBundle/Fixture/NewTshirtProductFixture.php [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+                    't_shirt_collection' => sprintf('Sylius %s %s', $this->faker->randomElement(['Summer', 'Winter', 'Spring', 'Autumn']), random_int(1995, 2012)),
+                    't_shirt_material' => $this->faker->randomElement(['Centipede', 'Wool', 'Centipede 10% / Wool 90%', 'Potato 100%']),
```
Comment (CoderMaggie) [inline src/Sylius/Bundle/CoreBundle/Fixture/NewTshirtProductFixture.php]:
```suggestion
                    't_shirt_material' => $this->faker->randomElement(['100% viscose', '100% cotton', '51% viscose, 29% polyester, 20% nylon', '95% polyester, 5% elastane', '100% linen']),
```

File: src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php [hunk 4]
```diff
@@ -269,7 +267,7 @@ private function createChannelPricings(ProductVariantInterface $productVariant,
         /** @var ChannelPricingInterface $channelPricing */
         $channelPricing = $this->channelPricingFactory->createNew();
         $channelPricing->setChannelCode($channelCode);
-        $channelPricing->setPrice($this->faker->randomNumber(3));
+        $channelPricing->setPrice($this->faker->numberBetween(100,10000));
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php]:
```suggestion
        $channelPricing->setPrice($this->faker->numberBetween(100, 10000));
```

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml [hunk 5]
```diff
@@ -220,3 +220,204 @@ sylius_fixtures:
...
+                                        configuration:
+                                            US_WEB:
+                                                amount: 100.00
+                                actions:
+                                    -   type: "order_fixed_discount"
+                                        configuration:
+                                            US_WEB:
+                                                amount: 10.00
+
+                new_tshirt_product:
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]:
```suggestion
                new_tshirt_product: ~
```

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml [hunk 6]
```diff
@@ -220,3 +220,204 @@ sylius_fixtures:
...
+
+                caps_product:
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]: As above

File: src/Sylius/Bundle/CoreBundle/Fixture/SimilarProductAssociationFixture.php [hunk 7]
```diff
@@ -106,7 +106,7 @@ protected function configureOptionsNode(ArrayNodeDefinition $optionsNode): void
     private function getAssociatedProductsAsArray(ProductInterface $owner): array
     {
         $products = $this->productRepository->findBy(['mainTaxon' => $owner->getMainTaxon()]);
-        $products = $this->faker->randomElements($products, 3);
+        $products = $this->faker->randomElements($products, 2);
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Fixture/SimilarProductAssociationFixture.php]: Why?

**Comment by lchrusciel** [comment]
Thanks, Adam! :tada:

**Pull request merged**